src/lib/tooltip/tooltip.ts

```diff
@@ -149,8 +169,12 @@ export class MdTooltip implements OnDestroy {
   /** The message to be displayed in the tooltip */
   @Input('mdTooltip') get message() { return this._message; }
   set message(value: string) {
-    this._message = value;
-    this._setTooltipMessage(this._message);
+    if (this._message) { this._unregisterA11yMessage(this._message); }
+
+    // If the message is not a string (e.g. number), convert it to a string and trim it.
+    this._message = value ? `${value}`.trim() : '';
```

Comment:
Might be a good idea to have a test for this case in particular (passing in a non-string to the message). We have another pending PR that deals with the same issue: https://github.com/angular/material2/pull/6146